extension/logstash/logstash-output-doris.gemspec

```diff
@@ -41,7 +41,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'mini_cache', ">= 1.0.0", "< 2.0.0"
   s.add_runtime_dependency "rest-client", '~> 2.1'
 
-  s.add_development_dependency 'logstash-devutils', '~> 2.0', '>= 2.0.3'
-  s.add_development_dependency 'sinatra', '~> 2.0', '>= 2.0.8.1'
-  s.add_development_dependency 'webrick', '~> 1.6'
+  s.add_development_dependency 'logstash-devutils', '~> 1.3'
```

Comment:
Why change version?

Reply:
This is the development environment version of the library and does not affect the runtime environment. The previous version had dependency conflicts.